Samples~/WebGLExample/Scripts/WebGLAvatarLoader.cs

```diff
@@ -0,0 +1,63 @@
+using ReadyPlayerMe.Core;
+using UnityEngine;
+using BodyType = ReadyPlayerMe.Core.BodyType;
+
+namespace ReadyPlayerMe.Samples
+{
+    [RequireComponent(typeof(WebFrameHandler))]
+    public class WebGLAvatarLoader : MonoBehaviour
+    {
+        private const string TAG = nameof(WebGLAvatarLoader);
+        private GameObject avatar;
+        private string avatarUrl = "";
+        private WebFrameHandler webFrameHandler;
+
+
+        private void Start()
+        {
+            webFrameHandler = GetComponent<WebFrameHandler>();
+            webFrameHandler.OnAvatarExport += HandleAvatarLoaded;
+            webFrameHandler.OnUserSet += HandleUserSet;
+            webFrameHandler.OnUserAuthorized += HandleUserAuthorized;
+        }
+
+        private void OnAvatarLoadCompleted(object sender, CompletionEventArgs args)
+        {
+            if (avatar) Destroy(avatar);
+            avatar = args.Avatar;
+            if (args.Metadata.BodyType == BodyType.HalfBody)
+            {
+                avatar.transform.position = new Vector3(0, 1, 0);
+            }
+        }
+
+        private void OnAvatarLoadFailed(object sender, FailureEventArgs args)
+        {
+            SDKLogger.Log(TAG, $"Avatar Load failed with error: {args.Message}");
+        }
+
+        public void HandleAvatarLoaded(string newAvatarUrl)
+        {
+            LoadAvatarFromUrl(newAvatarUrl);
+        }
+
+        public void HandleUserSet(string userId)
+        {
+            Debug.Log($"User set: {userId}");
+        }
+
+        public void HandleUserAuthorized(string userId)
+        {
+            Debug.Log($"User authorized: {userId}");
```

Comment:
I realize this is probably just copy /paste from the example repo but can yo u update to use SDKLogger.Log for logging. It means that devs can easily toggle on/off the logging from Core settings